backend/models/meta.js

```diff
@@ -30,18 +30,20 @@ const C = require("../constants");
 const utils = require("../utils");
 const systemLogger = require("../logger").systemLogger;
 
-function clean(metaListToClean) {
-	metaListToClean.forEach((metadataToClean) => {
-		if (metadataToClean._id) {
-			metadataToClean._id = utils.uuidToString(metadataToClean._id);
-		}
+function cleanOne(metadataToClean) {
```

Comment:
This is a bit confusing since in every other file clean() cleans a single object.

I think it's better to call this clean, and the one that takes in a list to call cleanMultiple, cleanMetaArray or something

Reply:
@Charence  this is resolved but not addresssed?